docs/contributing/sdk.md

```diff
@@ -0,0 +1,127 @@
+---
+id: sdk
+title: Contributing to XSOAR SDK
+slug: sdk
+authors: [kgal-pan]
+tags: [sdk,python,dev_env,contributing,pr]
+---
+
+# Contributing to XSOAR SDK
+
+Contributions to [`demisto-sdk`](https://github.com/demisto/demisto-sdk) are welcome and appreciated. 
+
+Some common ways to contribute are:
+
+- Create new commands.
+- Improve existing implementation.
+- Fix bugs.
+- Improve documentation. 
+
+### Prerequesites
+
+Make sure you already have the following requirements:
+
+* An active GitHub account.
+* A [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of the [`demisto-sdk`](https://github.com/demisto/demisto-sdk) repository. 
+* The forked repository cloned in your local machine or a codespace.
+* Have Python 3 and [`poetry`](https://python-poetry.org/docs/#installation) installed.
+* [Docker](https://docs.docker.com/engine/install/) installed.
+
+## How to Contribute
+
+Before contributing, we need to make sure we set up a development enviromment. Once that is ready, we can begin the development process which includes adding or modifying code, adding unit tests and documenting the changes. Upon completion, we can open a Pull Request to push those changes into the next release of the SDK. 
+
+### 1. Set Up Development Environment
+
+Run the following command from the repository root directory:
+
+1. Create a new branch to hold contributed work:
+
+```bash
+❯ git checkout -b $BRANCH_NAME
+```
+
+2. Install all required dependencies:
+
+```bash
+❯ poetry install
+```
+
+3. Activate the virtual environment:
+
+```bash
+❯ poetry shell 
+```
```

Comment:
need to switch 1 and 2, first its required to run poetry install against the master branch and only then checkout to a new branch once all the deps have been installed

current step 2 and step 3 should become step 1 and step 2.
current step 3 should become step 3